db/data/20200529093419_add_reporting_org_to_activities_where_missing.rb

```diff
@@ -0,0 +1,25 @@
+class AddReportingOrgToActivitiesWhereMissing < ActiveRecord::Migration[6.0]
+  def up
+    activities = Activity.where(reporting_organisation: nil)
+    service_owner = Organisation.find_by_service_owner(true)
+    activities.each do |activity|
+      creating_organisation = activity.organisation
+      reporting_organisation = case activity.level
+      when :fund
+        creating_organisation
```

Comment:
These always scramble my brain. I _think_ fund and programme always needs to have BEIS as the reporter, however project and sub-project can always be the organisation that created it? If so these should be flipped around.

Reply:
Looking at Meyrics mapping table, the reporter is always BEIS! https://miro.com/app/board/o9J_kv27qEM=/?moveToWidget=3074457347355940764&cot=13 What do we set for UKSA when they create a project on our latest version?

Reply:
I took the logic from the Create[Level]Service classes, in which case are they wrong? https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/app/services/create_fund_activity.rb#L11

Reply:
For projects it varies according to https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/app/services/create_project_activity.rb#L44

Reply:
This is to tidy up two very old activities which are otherwise entirely nil

Reply:
Understood 👍 As we know funds and programmes can only be created by BEIS, the BEIS org will only ever be the `creating_organisation` so it becomes equivalant to `service_owner`. I think this is fine as it'll only be run once 👍